From: Carlos Maddela <e7appew@gmail.com>
Date: Mon, 19 Dec 2016 09:39:49 +1100
Description: Make builds reproducible.
 Use the Debian changelog details to derive a build number and date,
 so that we can make reproducible binaries.
Forwarded: not-needed
Last-Update: 2013-09-22
---
diff --git a/scripts/time-stamper.bash b/scripts/time-stamper.bash
index fd28794..4ec3e60 100755
--- a/scripts/time-stamper.bash
+++ b/scripts/time-stamper.bash
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-build=$(grep BUILD $1  | cut -d\  -f3)
-build=$((build+1))
-echo "#define BUILD $build" >$1
+# Use Debian changelog details to derive build number and date.
+build=$(echo "${DEB_VERSION}" | cut -f2 -d-)
+echo "#define BUILD \"$build\"" >$1
